src/options/optionsMain.ts

```diff
@@ -115,6 +115,11 @@ export async function getOptions(rawOptions: any): Promise<AppOptions> {
     log.setLevel('info');
   }
 
+  if (rawOptions.wvvmp) {
```

Comment:
Please add an extra log here documenting that this uses an unofficial Electron build and you use it at your own risk.

Reply:
How does the following sound?
```ts
      log.warn(
        `\nATTENTION: Using the **unofficial** Electron for Content Security from castLabs`,
        "\nIt implements Google's Widevine Content Decryption Module (CDM) for DRM-enabled playback.",
        `\nSimply abort & re-run without passing the wvvmp flag to default to ${DEFAULT_ELECTRON_VERSION}`,
      );
```

Reply:
Awesome. Plus the warning about missing builds mentioned in the above comment, that would be:

```ts
      log.warn(
        `\nATTENTION: Using the **unofficial** Electron for Content Security from castLabs`,
        "\nIt implements Google's Widevine Content Decryption Module (CDM) for DRM-enabled playback.",
        "\nBe sure to ask for a version that is available at https://github.com/castlabs/electron-releases/releases !",
        `\nSimply abort & re-run without passing the wvvmp flag to default to ${DEFAULT_ELECTRON_VERSION}`,
      );
```

Reply:
:+1: